database/migrations/2022_07_22_030715_create_mentors_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
```

Comment:
name of migration??? Please follow Laravel standard

Reply:
fixed